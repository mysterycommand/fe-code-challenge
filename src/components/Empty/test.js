import React from 'react';
import Empty from './';

describe('Empty', () => {
  it('should be empty', () => {
    expect(<Empty />).toMatchSnapshot();
  });
});
