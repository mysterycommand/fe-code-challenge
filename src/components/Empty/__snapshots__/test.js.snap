// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty should be empty 1`] = `<Unknown />`;
